tasks/github_tasks.py

```diff
@@ -152,3 +152,16 @@ def _get_code_owners(root_folder):
                 # example /tools/retry_file_dump ['@DataDog/agent-metrics-logs']
                 owners[path] = parts[1:]
     return owners
+
+
+@task
+def get_milestone_id(_, milestone):
+    # Local import as github isn't part of our default set of installed
+    # dependencies, and we don't to propagate it to files importing this one
```

Comment:
```suggestion
    # dependencies, and we don't want to propagate it to files importing this one
```

Reply:
Since you managed to understand without the verb, is it really required? :grin: (fixed, thanks!)